package.json

```diff
@@ -37,12 +37,13 @@
   },
   "devDependencies": {
     "@actions/core": "1.2.6",
-    "@actions/github": "2.1.1",
+    "@actions/github": "4.0.0",
     "@types/node": "13.11.0",
-    "@zeit/ncc": "0.22.0",
+    "@vercel/ncc": "0.25.1",
     "eslint": "6.8.0",
     "eslint-config-neo": "0.5.2",
     "husky": "^4.2.3",
+    "is-plain-object": "^5.0.0",
```

Comment:
See https://github.com/octokit/rest.js/issues/1929 - was getting build errors for isPlainObject before this coming from octokit. Changing the tsconfig as suggested in that thread produced other errors so I decided on explicitly resolving this here instead.
ncc is also maintained by vercel now rather than zeit

Reply:
Changing the version of ncc is risky. Make sure you test the new build.

Reply:
I tested using a fork with a committed build as you suggested: https://github.com/Lewis65/ticket-check-action then I just replicated those changes in this branch.